src/fe-text/irssi.c

```diff
@@ -74,11 +75,20 @@ void mainwindow_activity_deinit(void);
 void mainwindows_layout_init(void);
 void mainwindows_layout_deinit(void);
 
+#ifdef HAVE_SECCOMP
```

Comment:
So the only ifdef should be this one?

if (sandbox)
#ifdef HAVE_SECCOMP
        ... do sandbox stuff
#else 
        ... tell the user sandboxing is not supported
#endif 
}